### ISSUE TITLE
lint: octal values 

### ISSUE THREAD

**Issue by cristaloleg** [go-critic/go-critic#435]
Before:
```go
foo(012) // might be a mistake
```
After:
```go
???
```

We should skip octal values passed to `os` package functions, 'cause it's a common practice to pass them as permission flags.

#VeryOpinionated

**Comment by Quasilyte** [comment]
Opinionated?

**Comment by cristaloleg** [comment]
As hell 😈

**Comment by Quasilyte** [comment]
I think it's quite trivial to do with `VisitExpr` and matching of `ast.BasicLit` with op equal to `token.INT` and value string starting from 0.
